### PR TITLE
Update to display stop type in tooltip

### PR DESCRIPTION
**Pull request by alvak1**
Testing instructions:

After login click "Create new disruptions" button. Fill the disruption page with details and proceed to selecting consequence type. Select consequence type stops or services. Search for services or stop to display the markers on the page. Hover over any marker to view the updated text.

### CODE REVIEW

File: site/utils/formUtils.ts [hunk 1]
```diff
@@ -72,3 +72,23 @@ export const getDataInPages = <T>(pageNumber: number, data: T[]): T[] => {
     const endPoint = pageNumber * 10;
     return data.slice(startPoint, endPoint);
 };
+
+export const getStopType = (stopType: string | undefined) => {
+    let stop = "";
```
Comment (alainajaffer) [inline site/utils/formUtils.ts:77]: you can remove this and just do return "Bus stop" on like 80 same for the others and then remove line 93. The breaks can then be removed also

Comment (alvak1) [reply site/utils/formUtils.ts:77]: done changed to if statements

**Review by alainajaffer — APPROVED** [review]

File: site/utils/formUtils.ts [hunk 2]
```diff
@@ -72,3 +72,15 @@ export const getDataInPages = <T>(pageNumber: number, data: T[]): T[] => {
...
     return data.slice(startPoint, endPoint);
 };
+
+export const getStopType = (stopType: string | undefined) => {
+    if (stopType === "BCT") {
+        return "Bus stop";
+    } else if (stopType === "MET" || stopType === "PLT") {
+        return "Tram stop";
+    } else if (stopType === "FER" || stopType === "FBT") {
+        return "Tram stop";
```
Comment (lj-kpmg) [inline site/utils/formUtils.ts:82]:
```suggestion
        return "Ferry terminal";
```

**Review by lj-kpmg — CHANGES_REQUESTED** [review]
Tram stop needs changing to Ferry terminal for FER and FBT types

File: site/utils/formUtils.ts (hunk 2 above)
Comment (alvak1) [reply site/utils/formUtils.ts]: done

**Review by lj-kpmg — APPROVED** [review]

**Pull request merged**